dd-java-agent/instrumentation/jetty-6/src/main/java/datadog/trace/instrumentation/jetty6/JettyHandlerAdvice.java

```diff
@@ -0,0 +1,95 @@
+// Modified by SignalFx
+package datadog.trace.instrumentation.jetty6;
+
+import static datadog.trace.instrumentation.jetty6.JettyDecorator.DECORATE;
+
+import datadog.trace.api.DDTags;
+import datadog.trace.context.TraceScope;
+import io.opentracing.Scope;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.propagation.Format;
+import io.opentracing.tag.Tags;
+import io.opentracing.util.GlobalTracer;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import net.bytebuddy.asm.Advice;
+import org.mortbay.jetty.HttpConnection;
+import org.mortbay.jetty.Response;
+
+public class JettyHandlerAdvice {
+
+  @Advice.OnMethodEnter(suppress = Throwable.class)
+  public static Scope startSpan(
+      @Advice.This final Object source, @Advice.Argument(1) final HttpServletRequest req) {
+
+    System.out.println("Starting 6 span");
+
+    if (GlobalTracer.get().activeSpan() != null) {
+      // Tracing might already be applied.  If so ignore this.
+      return null;
+    }
+
+    final SpanContext extractedContext =
+        GlobalTracer.get()
+            .extract(Format.Builtin.HTTP_HEADERS, new HttpServletRequestExtractAdapter(req));
+    final Scope scope =
+        GlobalTracer.get()
+            .buildSpan("jetty.request")
+            .asChildOf(extractedContext)
+            .withTag("span.origin.type", source.getClass().getName())
+            .startActive(false);
+
+    final Span span = scope.span();
+    final String resourceName = req.getMethod() + " " + source.getClass().getName();
```

Comment:
We no longer need the method in the operation name

Reply:
Updated in #37